lib/resty/cassandra/policies/lb/req_dc_rr.lua

```diff
@@ -47,7 +51,11 @@ function _M:init(peers)
 
   for i = 1, #peers do
     if type(peers[i].data_center) ~= 'string' then
-      error('peer '..peers[i].host..' data_center field must be a string')
+      log(WARN, _log_prefix, 'peer ', peers[i].host, ' data_center field must be a string')
```

Comment:
A warning should be more descriptive: what is the impact of this warning? What is being done by the software to remediate?

> peer ... has no data_center field in shm therefore considered a remote peer

Just a an example to be improved upon. Precise, informative, concise is the standard to aim for.